src/Data.h

```diff
@@ -15,6 +15,10 @@ namespace TW {
 using byte = std::uint8_t;
 using Data = std::vector<byte>;
 
+inline void pad_left(Data& data, const uint32_t size) {
+    data.insert(data.begin(), size - data.size(), 0);
```

Comment:
do we need to check `size - data.size()`?

Reply:
Yes, we have to add `size - data.size()` zeros to the vector. If you have a better way to do it, let me know.